src/App.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import App from './App';
+
+import locations from '../fixtures/locations';
+import categories from '../fixtures/categories';
+import { useDispatch, useSelector } from '../__mocks__/react-redux';
```

Comment:
```suggestion
import { useDispatch, useSelector } from 'react-redux';
```

이미 jest.mock을 통해서 해당 모듈들은 __mocks__/react-redux에 있는 모듈들로 대체하도록 처리했기 때문에 위와 같이 임포트하는게 맞을 것 같아요~